.github/workflows/publish-to-aws.yml

```diff
@@ -0,0 +1,25 @@
+name: Publish to AWS
+
+on:
+  workflow_dispatch:
+
+jobs:
+  publish_to_digitalocean:
+    runs-on: ubuntu-latest
+    name: Publish image to AWS
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Install Packer
+      run: |
+        curl -fsSL https://apt.releases.hashicorp.com/gpg | sudo apt-key add -
+        sudo apt-add-repository "deb [arch=amd64] https://apt.releases.hashicorp.com $(lsb_release -cs) main"
+        sudo apt-get update && sudo apt-get install packer
+        packer --version
+    - name: Create Image
+      run: |
+        cd ./.build/aws
+        export AWS_ACCESS_KEY_ID=${{ secrets.AWS_ACCESS_KEY_ID }}
+        export AWS_SECRET_ACCESS_KEY=${{ secrets.AWS_SECRET_ACCESS_KEY }}

```

Comment:
needs to be added to the secrets and set up following https://learn.hashicorp.com/tutorials/packer/aws-get-started-build-image?in=packer/aws-get-started#authenticate-to-aws

Reply:
hopefully done